Modules/vc-module-catalog/VirtoCommerce.CatalogModule.Web/Module.cs

```diff
@@ -231,7 +243,15 @@ public void PostInitialize(IApplicationBuilder appBuilder)
                         nameof(ExportableProduct.Images)))
                     .WithTabularMetadata(typeof(ExportableProduct).GetPropertyNames()));
 
+            var catalogFullExportPagedDataSourceFactory = appBuilder.ApplicationServices.GetService<Func<ExportDataQuery, CatalogFullExportPagedDataSource>>();
+            registrar.RegisterType(
+                ExportedTypeDefinitionBuilder.Build<ExportableCatalogFull, CatalogFullExportDataQuery>()
+                    .WithDataSourceFactory(appBuilder.ApplicationServices.GetService<CatalogFullExportPagedDataSourceFactory>())
+                    .WithMetadata(new ExportedTypeMetadata { PropertyInfos = new ExportedTypePropertyInfo[] { } })
+                    );
+
             AbstractTypeFactory<ExportDataQuery>.RegisterType<ProductExportDataQuery>();
+            AbstractTypeFactory<ExportDataQuery>.RegisterType<CatalogFullExportDataQuery>();
```

Comment:
No need in these registrations too, they are done in ExportedTypeDefinitionBuilder now.

Reply:
As all the changes below.

Reply:
OK